syntax = "proto3";

package helloworld;

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
  rpc SayHelloAgain (HelloRequest) returns (HelloReply) {} // Nuevo m√©todo
}

message HelloRequest {
  string name = 1;
}

message HelloReply {
  string message = 1;
}
